docs/users/forwarding.md

```diff
@@ -0,0 +1,61 @@
+# Configuring player information forwarding
+
+Velocity supports forwarding information about your players to your servers, such as IP addresses, UUIDs and skins.
+Velocity supports two different methods for forwarding player information to your servers, which are described in the
+ appropriate sections.
+ 
+## Configuring modern forwarding
+`modern` forwarding is a Velocity-native format. It forwards all player information in an efficient binary format and
+employs a MAC code to make it much more difficult to trick the server into impersonating your Velocity proxy.
+However, it is only available for Minecraft 1.13 or higher.
+
+To use modern forwarding with any supported server implementation, set the `player-info-forwarding` setting in
+`velocity.toml` to `modern`. You must also change the `forwarding-secret` setting to a unique secret. You then need
+to ensure your server is properly configured to use Velocity forwarding. 
+
+```css
```

Comment:
Can we do better here?

Reply:
I have changed this in d6ec3c6779f859bfc809b0dbb3930a7da0edf052, let me know if you'd like another revision.

Reply:
The wording is improved somewhat, but there has to be a better way to display this warning.